src/app/app.component.ts

```diff
@@ -34,6 +34,7 @@ export class AppComponent implements OnInit {
   ngOnInit() {
     this.keepAlive();
     this.checkAngularProject();
+    this.loadCurrentServe();
```

Comment:
flow:
1. `keepAlive`
2. `loadCurrentServe`:
      true -> go to serve screen.
      false -> `checkAngularProject`